src/main/rx/cc2500_frsky_d.c

```diff
@@ -222,6 +222,7 @@ rx_spi_received_e frSkyDDataReceived(uint8_t *packet)
 
             break;
         }
+        FALLTHROUGH; //!!TODO -check this fall through is correct
```

Comment:
Seems to be correct (if not exactly pretty). `STATE_UPDATE` is entered when the RX is ready for the next frame. It then goes into `STATE_DATA` where it waits until the frame has been received.